test/analytics/tests/analytics_from_the_browser.ts

```diff
@@ -24,7 +24,11 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       return await browser.execute(
         ({ takeNumberOfCounters }) =>
           window.__analyticsPluginA__.stats
-            .filter((counter) => counter.event_type === 'test-plugin-lifecycle')
+            .filter(
+              (counter) =>
+                counter.event_type === 'test-plugin-lifecycle' &&
+                ['client', 'FTR-shipper'].includes(counter.source)
+            )
```

Comment:
simplify the tests by removing the effect in the `telemetryCounters$` from other shippers.